data/en/mbs_0251.json

```diff
@@ -0,0 +1,703 @@
+{
+    "title": "Monthly Business Survey",
+    "survey_id": "009",
+    "mime_type": "application/json/ons/eq",
+    "theme": "default",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.1",
+    "legal_basis": "StatisticsOfTradeAct",
+    "description": "mbs",
+    "form_type": "0251",
+    "view_submitted_response": {
+        "enabled": true,
+        "duration": 900
+    },
+    "sections": [{
+            "id": "section",
+            "groups": [{
+                "id": "group",
+                "title": "",
+                "blocks": [{
+                        "type": "Introduction",
+                        "id": "introduction",
+                        "primary_content": [{
+                            "type": "Basic",
+                            "id": "use-of-information",
+                            "content": [{
+                                "list": [
+                                    "Data should relate to all sites in England, Scotland, Wales and Northern Ireland.",
```

Comment:
take out Northern Ireland same as 0201

Reply:
Not in this list so think we should leave it.
https://trello.com/c/WWan8FIg